searx/engines/duckduckgo_extra.py

```diff
@@ -57,9 +57,12 @@ def request(query, params):
         # 'u': 'bing',
         'l': eng_region,
         'f': ',,,,,',
-        'vqd': get_vqd(query),
     }
 
+    vqd = get_vqd(query)
+    if vqd != '':
+        args[vqd] = vqd
+
```

Comment:
Not tested but I assume here we should do the same as in WEB .. don't try to send a request to DDG if we do not have a vqd:

```python
    vqd = get_vqd(query)

    if vqd == '':
        # some search terms do not have results and therefore no vqd value
        params['url'] = None
        return params
```

Reply:
done / see [diff](https://github.com/searxng/searxng/compare/05aaabd485cb9d631c24af817999b1b0656a071e..2b11f300ca99a0f1cb1de9182a50c42db0d1b3fb)